support-frontend/assets/helpers/page/page.ts

```diff
@@ -1,19 +1,15 @@
 // ----- Imports ----- //
 import type { Participations } from 'helpers/abTests/abtest';
-import * as abTest from 'helpers/abTests/abtest';
-import { Country, CountryGroup } from 'helpers/internationalisation';
+import { Country } from 'helpers/internationalisation';
 import type { IsoCountry } from 'helpers/internationalisation/country';
-import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
 import {
 	analyticsInitialisation,
 	consentInitialisation,
 } from 'helpers/page/analyticsAndConsent';
 import { getReferrerAcquisitionData } from 'helpers/tracking/acquisitions';
 
-function setUpTrackingAndConsents(): void {
+function setUpTrackingAndConsents(participations: Participations = {}): void {
```

Comment:
Actually while pairing with George, I tried to use -function setUpTrackingAndConsents(participations?: Participations = {} , but it started showing  error , -
<img width="938" alt="image" src="https://github.com/guardian/support-frontend/assets/73653255/f9c79390-8844-4cb5-b112-e58972c54555">

So changes it as per the solution [here](https://stackoverflow.com/questions/17186566/how-do-i-fix-error-ts1015-parameter-cannot-have-question-mark-and-initializer#:~:text=We%20can%20see%20see%20that,optional%20parameter%20as%20you%20intend.)

Reply:
The Typescript error was down to assigning a default value to an optional parameter.

We can make the `participations` parameter optional, but we'll then have to either make `participations` optional further downstream (eg. in `analyticsInitialisation`), Or we conditionally pass `participations` if defined OR `{}` if  `participations` is `undefined` to  `analyticsInitialisation`?

I think we decided to settle on passing a fallback default value `{}` as we wouldn't need to update all downstream references to `participations` to handle it being optional.

Reply:
Gotcha. As per [the docs](https://www.typescriptlang.org/docs/handbook/2/functions.html#optional-parameters)

> Optional parameters... by marking the parameter as optional with `?`... You can also provide a parameter default

Reply:
If you see my other comment https://github.com/guardian/support-frontend/pull/5565/files/25ac22fa0241dfff606000de7e8869b6c8a79c83#r1431620202, I don't think this change should be in this PR, I believe the changes we want to merge are all in the 1st commit, so I'd recommend we revert the 2nd commit. 

The changes in this 2nd commit are the initial changes implemented to refactor this logic, but this can be done in a follow-up PR.